### ISSUE TITLE
student loss fine tuning

### ISSUE THREAD

**Issue by zhanghanqi** [anubhavashok/N2N#11]
I use cifar10_old and resnet18 to achieve acc=92.01% .After stage1 training ,the best acc is 85.81%.The model_6_2 gets the best result.I see your answer of the other questions. I know Student Loss need to change.  Whether it's this loss = MSEloss(studentOutput, teacherOutput) in this funtion trainTeacherStudentParallel that needs to change.    Should it be changed into loss=MSEloss + rc*ra

**Comment by anubhavashok** [comment]
It's best to use loss = MSEloss(studentOutput, teacherOutput) +\lambda CrossEntropyLoss(studentOutput, target). The optimal value of lambda can vary based on the model produced. Something that also works well is training with MSELoss(studentOutput, teacherOutput) for a few epochs and then finetuning with CrossEntropyLoss(studentOutput, target).

rc & ra are reward terms used strictly for the RL architecture search part.

**Comment by zhanghanqi** [comment]
Thank you very much! I will have a try.
